### ISSUE TITLE
Implementation of Data Validation

### ISSUE THREAD

**Issue by ClemensBeck** [slub/dlf-3d-viewers#41]
* [x] @markusweigelt Implementation of validation for: 
- scheme
- xml formats
- supporting of various application profiles

Steps to achieve: 
* [x] Finishing application profile (see #25)
* [ ] Implementation validiation mechanism (see https://github.com/slub/dlf-3d-viewers/issues/75)

**Comment by ClemensBeck** [comment]
Should be done in the next sprint (beginning in August)

**Comment by markusweigelt** [comment]
This is too large a issue to complete within a single sprint. Maybe we should split this into multiple issues. Firstly, the Application Profile document needs to be finalized. Then, the technical implementation as an XML Schema must be done. Lastly, a validation mechanism needs to be implemented. This entire process will take considerable time and multiple development iterations to achieve the desired result.

**Comment by ClemensBeck** [comment]
@markusweigelt I agree with you. I will split it in several smaller issues. The finishing of the application profile should be done in the end of the sprint, hopefully by Sebastian.
